### ISSUE TITLE
Document bootstrapper extensibility

### ISSUE THREAD

**Issue by tmat** [dotnet/arcade#1456]
Missing:
`restore-toolset.ps1`
`configure-toolset.ps1`

**Comment by jcagme** [comment]
By documenting you mean adding the scripts in the description?

**Comment by hughbe** [comment]
Yeah would be nice to document these scripts and how to integrate them. For example, in winforms we called `init-tools-native.ps1`: https://github.com/dotnet/winforms/pull/3335
